pyartcd/pyartcd/pipelines/build_sync.py

```diff
@@ -36,8 +37,61 @@ def __init__(self, runtime: Runtime, version: str, assembly: str, publish: bool,
         self.logger = runtime.logger
         self.working_dir = self.runtime.working_dir
         self.fail_count_name = f'{version}-assembly.{assembly}.count'
+        self.job_run = self.runtime.get_job_run_url()
+
+    async def comment_on_assembly_pr(self, text_body):
+        """
+        Comment the link to this jenkins build on the assembly PR if it was triggered automatically
+        """
+        # Keeping in try-except so that job doesn't fail because of any error here
+        try:
+            base_repo = "openshift-eng"
+            repository = "ocp-build-data"
+            branch = self.doozer_data_gitref
```

Comment:
So the param `doozer_data_gitref` may not always be given when triggering a job. 
Doozer's behavior is: it checks if the doozer_data_gitref is given then it chooses that, if not
then it sets the branch to "openshift-{major}.{minor}". So we should do the same here

Reply:
added in https://github.com/openshift-eng/aos-cd-jobs/pull/3892/commits/545b6f0d09a4124236d83c23f986d1dacf4c126e

Reply:
Great!